src/main/java/com/microsoft/sqlserver/jdbc/SQLServerSpatialDatatype.java

```diff
@@ -1662,14 +1662,30 @@ private void skipWhiteSpaces() {
         this.pointOffset = pointOffset;
     }
 
+    /**
+     * Returns the figuresAttribute value.
```

Comment:
I am changing everything to be consistent. For getters, can you please say "Gets blah blah"?